scripts/incentive-spreadsheet-registry.ts

```diff
@@ -63,4 +63,13 @@ export const FILES: { [ident: string]: IncentiveFile } = {
     enHeader: 'Program Description (guideline)',
     esHeader: 'Program Description (Spanish)',
   },
+  WI: {
+    filepath: 'data/WI/incentives.json',
+    sheetUrl:
+      'https://docs.google.com/spreadsheets/d/e/2PACX-1vSxNnovJ_viGErGJIkLL8uyXNx6QWKSgLZlntceubHXeYbb_s3Ksjxgg6ZSU8MiayjtmaMUpiHC1ue9/pub?gid=995688950&single=true&output=csv',
+    headerRowNumber: 2,
+    idHeader: 'ID',
+    enHeader: 'Program Description (style guide)',
+    runSpreadsheetHealthCheck: true,
```

Comment:
In order to confirm this actually passes the test, you need to temporarily flip the `skip` boolean in `test/data/spreadsheets-health.test.ts` to false. The test is skipped by default.

If it doesn't pass, that's fine, just remove this line from the spreadsheet registry. This test is evolving fast anyway so I wouldn't invest a lot of time making it pass.

Either way, revert the skip boolean when you're done.